platform/ui/src/components/ContextMenu/ContextMenuController.tsx

```diff
@@ -0,0 +1,213 @@
+import * as ContextMenuItemsBuilder from './ContextMenuItemsBuilder';
+import ContextMenu from './ContextMenu';
+import { CommandsManager, ServicesManager, Types } from '@ohif/core';
+import { Menu, MenuItem, Point, ContextMenuProps } from './types';
+
+/**
+ * The context menu controller is a helper class that knows how
+ * to manage context menus based on the UI Customization Service.
+ * There are a few parts to this:
+ *    1. Basic controls to manage displaying and hiding context menus
+ *    2. Menu selection services, which use the UI customization service
+ *       to choose which menu to display
+ *    3. Menu item adapter services to convert menu items into displayable and actionable items.
+ *
+ * The format for a menu is defined in the exported type MenuItem
+ */
+export default class ContextMenuController {
```

Comment:
This file has a load of logic based on various `services`, I belive we should keep separating the ui from the business logic as much as we can. The `platform/ui` should only be react and react based ui components, the reset of logic should be injected (wrapped, etc.) somewhere else

Reply:
Moved as requested.